Toolset/palettes/tutorial/courses/First Run/tutorials/BMI Calculator/lessons/App Lesson.txt

```diff
@@ -1338,25 +1484,38 @@ action
 end step
 
 step "Set Calculator Header Script"
-	Set the script of the header to the following (you can delete the
-	leftAction and searchAction handlers as they will not be used by 
-	this widget):
+	Set the script of the header to the following, and click the apply
+	button:
```

Comment:
I think you need a full stop rather than a colon.  Also, should "apply" be capitalised/quoted?